src/app/component/comments/services/comments.service.ts

```diff
@@ -29,4 +29,8 @@ export class CommentsService {
   public getCommentsByPage(): Observable<object> {
     return this.http.get(`https://greencity.azurewebsites.net/econews/comments?ecoNewsId=${this.ecoNewsId}&page=0&size=12`);
   }
+
+  public deleteComments(id) {
+    return this.http.delete(`https://greencity.azurewebsites.net/econews/comments?id=${id}`, {observe: 'response'});
```

Comment:
Please use alias for part of link line 34. We have this alias.

Reply:
fixed